modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-accessing-your-cluster-quick"]
+= Accessing your cluster
+
+To log in to your cluster, you can use this quick access procedure.
+
+
+[NOTE]
+====
+As a best practice, xref:../getting_started_rosa/accessing-rosa-cluster.html#rosa-accessing-your-cluster[access your cluster with an IDP account] instead.
+====
+
+
+.Procedure
+
+. Enter the following command:
++
+[source, terminal]
```

Comment:
We're omitting the space here:

`[source,terminal]`

Reply:
Done.